app/helpers/locations_helper.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module LocationsHelper
+  def location_display_name(school, show_postcode: false, show_urn: false)
```

Comment:
```suggestion
  def location_display_name(location, show_postcode: false, show_urn: false)
```

I would suggest changing this too, but the method name looks good now.